apps/web/src/views/Swap/V3Swap/containers/SwapCommitButton.tsx

```diff
@@ -1,42 +1,46 @@
 import { useTranslation } from '@pancakeswap/localization'
 import { TradeType } from '@pancakeswap/sdk'
-import { Button, Text, useModal, Box, Message, MessageText, AutoColumn, Dots } from '@pancakeswap/uikit'
+import { SmartRouterTrade } from '@pancakeswap/smart-router/evm'
+import { CurrencyAmount, Fraction, ONE, Percent } from '@pancakeswap/swap-sdk-core'
+import { AutoColumn, Box, Button, Dots, Message, MessageText, Text, useModal } from '@pancakeswap/uikit'
+import { UNIVERSAL_ROUTER_ADDRESS } from '@pancakeswap/universal-router-sdk'
+import { useExpertMode } from '@pancakeswap/utils/user'
 import { confirmPriceImpactWithoutFee } from '@pancakeswap/widgets-internal'
-import { useCallback, useEffect, useState, useMemo, memo } from 'react'
-import { SMART_ROUTER_ADDRESSES, SmartRouterTrade } from '@pancakeswap/smart-router/evm'
+import { memo, useCallback, useEffect, useMemo, useState } from 'react'
 import { logGTMClickSwapEvent } from 'utils/customGTMEventTracking'
 
-import { useIsTransactionUnsupported } from 'hooks/Trades'
+import { PERMIT2_ADDRESS } from '@pancakeswap/permit2-sdk'
```

Comment:
need to add multichain support

Reply:
Permit2Address should be the same cross chain, if create account and nonce is same

Reply:
for most evm chains it is, but probably not with zksync. Let's double check with sc devs